### PR TITLE
Scheduling Profiler marks should include thrown Errors

### PR DESCRIPTION
**Pull request by bvaughn**
Builds on top of #22417 and adds marks for thrown Errors to the Scheduling Profiler

error-marks

### CODE REVIEW

File: packages/react-devtools-scheduling-profiler/src/CanvasPage.js [hunk 1]
```diff
@@ -461,14 +481,7 @@ function AutoSizedCanvas({
...
           setHoveredEvent({
-            componentMeasure: null,
-            flamechartStackFrame: null,
-            measure: null,
-            nativeEvent: null,
-            networkMeasure: null,
-            schedulingEvent: null,
-            snapshot: null,
-            suspenseEvent: null,
+            ...EMPTY_CONTEXT_INFO,
```
Comment (bvaughn) [inline packages/react-devtools-scheduling-profiler/src/CanvasPage.js:484]: All of the repetition in this file was driving me crazy so I cleaned it up a bit with this change.

File: packages/react-devtools-shared/src/constants.js [hunk 2]
```diff
@@ -316,9 +318,11 @@ export const THEME_STYLES: {[style: Theme | DisplayDensity]: any} = {
     '--color-scheduling-profiler-react-suspense-resolved-hover': '#89d281',
     '--color-scheduling-profiler-react-suspense-unresolved': '#c9cacd',
     '--color-scheduling-profiler-react-suspense-unresolved-hover': '#93959a',
+    '--color-scheduling-profiler-thrown-error': '#fb3655',
+    '--color-scheduling-profiler-thrown-error-hover': '#f82042',
     '--color-scheduling-profiler-text-color': '#282c34',
     '--color-scheduling-profiler-text-dim-color': '#555b66',
-    '--color-scheduling-profiler-react-work-border': '#ffffff',
+    '--color-scheduling-profiler-react-work-border': '#3d424a',
```
Comment (bvaughn) [inline packages/react-devtools-shared/src/constants.js:325]: These color values were broken before. Noticed while implementing this so I fixed them too.

File: packages/react-devtools-scheduling-profiler/src/content-views/ReactMeasuresView.js [hunk 3]
```diff
@@ -136,15 +136,23 @@ export class ReactMeasuresView extends View {
...
+                width:
+                  timestampToPosition(
+                    nextMeasure.timestamp,
+                    scaleFactor,
+                    frame,
+                  ) - x,
+                height: REACT_MEASURE_HEIGHT,
+              },
+            };
+          }
```
Comment (bvaughn) [inline packages/react-devtools-scheduling-profiler/src/content-views/ReactMeasuresView.js:155]: Notice this overflow bug while I was testing profiling data sets, so I fixed it too. It's unrelated to the main change.

**Review by lunaruan — APPROVED** [review]
I have complete faith in you :D :D :D

**Pull request merged**